subsys/bluetooth/audio/bap_stream.c

```diff
@@ -161,11 +160,13 @@ enum bt_bap_ascs_reason bt_audio_verify_qos(const struct bt_codec_qos *qos)
 		return BT_BAP_ASCS_REASON_SDU;
 	}
 
+#if defined(CONFIG_BT_BAP_BROADCAST_SOURCE) || defined(CONFIG_BT_BAP_UNICAST)
```

Comment:
Coud `IS_ENABLED` be used?

Reply:
We could use `IS_ENABLED` instead of `defined` for `#if`, but that will result in the same. 

`IS_ENABLED` is primarily meant for `if` statements, and not the preprocessor `#if` as we cannot do something like `#IS_ENABLED`